iothub/device/src/TransportSettings/IotHubClientMqttSettings.cs

```diff
@@ -58,25 +45,6 @@ public IotHubClientMqttSettings(IotHubClientTransportProtocol transportProtocol
         /// </summary>
         public QualityOfService ReceivingQoS { get; set; }
 
-        /// <summary>
-        /// The maximum no. of inbound messages that are read from the channel.
-        /// The default value is 50.
-        /// </summary>
-        public int MaxPendingInboundMessages { get; set; }
```

Comment:
iirc this was used to set the client queue size for c2d messages. Is this no longer configurable?

Reply:
MQTTNet does appear to allow us to configure this, but I don't know of any use cases for configuring this value. We could add it back in later if a customer asks for it